src/Sulu/Bundle/SearchBundle/Controller/SearchController.php

```diff
@@ -65,27 +65,32 @@ public function __construct(
      */
     public function searchAction(Request $request)
     {
-        $query = $request->query->get('q');
+        $queryString = $request->query->get('q');
         $category = $request->query->get('category', null);
         $locale = $request->query->get('locale', null);
 
-
         $page = $this->listRestHelper->getPage();
         $limit = $this->listRestHelper->getLimit();
+        $aggregateHits = array();
+        $startTime = microtime(true);
 
-        $query = $this->searchManager->createSearch($query);
+        foreach ($this->searchManager->getIndexNames() as $indexName) {
+            $query = $this->searchManager->createSearch($queryString)->indexes(array($indexName));
 
-        if ($locale) {
-            $query->locale($locale);
-        }
+            if ($locale) {
+                $query->locale($locale);
+            }
+
+            if ($category) {
+                $query->category($category);
+            }
 
-        if ($category) {
-            $query->category($category);
+            $aggregateHits += $query->execute();
         }
 
-        $hits = $query->execute();
+        $time = microtime(true) - $startTime;
```

Comment:
It does matter to developers, and the API is for developers. Google also shows the query time (`About 61,400,000 results (0.62 seconds) `), so there is also a use case.